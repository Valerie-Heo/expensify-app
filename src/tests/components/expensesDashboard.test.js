import React from "react";
import { shallow } from "enzyme";
import ExpenseDashboardPage from "../../components/ExpenseDashboardPage";

test("this should render dashboard page", () => {
  const render = shallow(<ExpenseDashboardPage />);
  expect(render).toMatchSnapshot();
});
