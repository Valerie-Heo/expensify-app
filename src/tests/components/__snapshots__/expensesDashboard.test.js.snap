// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`this should render dashboard page 1`] = `
<div>
  <ExpenseSummary />
  <Connect(ExpenseListFilters) />
  <Connect(ExpenseList) />
</div>
`;
